src/CC-PDM-1.0.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-PDM-1.0" name="Creative
+   Commons Public Domain Mark 1.0 Universal" listVersionAdded="3.26.0">
+      <crossRefs>

```

Comment:
Would suggest adding https://creativecommons.org/publicdomain/mark/1.0/ here, since the CC site lists that as the "Canonical URL".